cmd/stack_lint_dockerfile_stack.go

```diff
@@ -77,9 +78,16 @@ func lintDockerFileStack(log *LoggingConfig, stackPath string) (int, int) {
 			if k == optionalEnvironmentVariables[i] {
 				variableFound = true
 			}
+			if k == "APPSODY_DEBUG" {
+				appsodyDebugFound = true
+			}
 		}
 		if !variableFound {
-			log.Warning.log("Missing ", variable)
+			errMsg := ""
+			if appsodyDebugFound == true && optionalEnvironmentVariables[i] == "APPSODY_DEBUG_PORT" {
```

Comment:
you can just do `if appsodyDebugFound` instead of `== true`, sorry for the nitpick

Reply:
Np, i think the go linter fails (the irony) with that anyway